docs/user/reference/resource-compute/list-of-commands-available-on-a-compute-resource-provisioned-by-juju/list-of-juju-introspect-macros/juju_agent.md

```diff
@@ -1,4 +1,42 @@
 (juju_agent)=
 # `juju_agent`
 
-TBA
+```{toctree}
+:hidden:
+
+list-of-juju-introspect-macros/index
+
+```
+
+## Summary
+
+Juju agent selector for introspect command. 
+
+## Usage
+
+```text
+juju-agent [command]                                  
```

Comment:
Here and everywhere else: Is it with a dash or an underscore? (I see `juju_engine_report` uses underscores.)

Reply:
Nice catch! It was `juju_agent`, you were right. Fixed everywhere.